hugo/content/_index.html

```diff
@@ -24,12 +24,12 @@ <h2>Multiple cursors</h2>
             <p>See what others are looking at: the cursors and selections of all participants are highlighted in your editor.</p>
         </div>
         <div class="content-card">
-            <h2>VS Code extension</h2>
-            <p>Get Open Collaboration Tools from <a href="https://open-vsx.org/extension/typefox/open-collaboration-tools">Open VSX</a> or the <a href="https://marketplace.visualstudio.com/items?itemName=typefox.open-collaboration-tools">VS Code Marketplace</a>.</p>
+            <h2>End-to-end encryption</h2>
+            <p>Messages between participants are encrypted so not even the server knows what's in your documents – only those you invited do.</p>
```

Comment:
Suggestion, the (directly) is optional: 
```suggestion
            <p>Messages are encrypted (directly) between participants so not even the server knows what you are sharing – only those you invited do.</p>
```